src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -5155,6 +5201,12 @@ namespace Akka.Util.Internal
         public static System.Text.StringBuilder AppendJoin<T>(this System.Text.StringBuilder sb, string separator, System.Collections.Generic.IEnumerable<T> values) { }
         public static System.Text.StringBuilder AppendJoin<T>(this System.Text.StringBuilder sb, string separator, System.Collections.Generic.IEnumerable<T> values, System.Action<System.Text.StringBuilder, T, int> valueAppender) { }
     }
+    public class static TaskEx
```

Comment:
@alexvaluyskiy that class conflicts with the built-in `System.Threading.Tasks.TaskExtensions`, which required me to use a ton of namespace aliases everywhere. Coming up with a class name with no collisions with a built-in type was the better choice and I'd move any existing methods on the old `TaskExtensions` class into `TaskEx` in the future.